docs/flow/reference/config-blocks/logging.md

```diff
@@ -6,3 +6,63 @@ weight: 100
 ---
 
 # `logging` block
+
+`logging` is an optional configuration block used to customize how Grafana
+Agent produces log messages. `logging` is specified without a label and can
+only be provided once per configuration file.
+
+> **NOTE**: Configuration blocks are not components, so expressions which
+> reference the exports of components may not be used.
+
+## Example
+
+```river
+logging {
+  level  = "info"
+  format = "logfmt"
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`level` | `string` | Level at which log lines should be written | `"info"` | no
+`format` | `string` | Format to use for writing log lines | `"logfmt"` | no
+
+### Log level
+
+The following log levels are recognized:
+
+* `error`: Only write logs at the "error" level.
+* `warn`: Only write logs at the "warn" level or above.
+* `info`: Only write logs at "info" level or above.
+* `debug`: Write all logs, including "debug" level logs.
+
+### Log format
+
+The following log line formats are recognized:
+
+* `logfmt`: Write logs as [logfmt][] lines.
+* `json`: Write logs as JSON objects.
+
+[logfmt]: https://brandur.org/logfmt
+
+## Log location
+
+Grafana Agent writes all logs to `stderr`.
+
+When running Grafana Agent as a systemd service, logs written to `stderr` can
+be viewed through `journald`.
+
+When running Grafana Agent as a container, logs written to `stderr` can be
+viewed through `docker logs` or `kubectl logs`, depending on if Docker or
+Kubernetes was used for deploying the agent.
+
+When running Grafana Agent as a Windows service, logs are instead written as
+event logs and can be viewed through Event Viewer.
```

Comment:
nit: add "Application event logs"

Reply:
Like this?

> When running Grafana Agent as a Windows service, logs are instead written as
application event logs and can be viewed through Event Viewer.